packages/backend/src/controllers/authentication.ts

```diff
@@ -73,6 +76,150 @@ export const getOpenIdUserFromRequest = ({
     return undefined;
 };
 
+const createOpenIdUser = (
```

Comment:
this makes more sense as two separate functions. One for openid userinfo objects and one for passport profiles. Currently this function does 2 distinct things

Reply:
Agree, checking how to separate them